readme.md

```diff
@@ -279,7 +279,7 @@ Thanks for contributing! 🦋🙌
 ### Editing pull requests
 
 - [](# "sync-pr-commit-title") 🔥 [Uses the PR’s title as the default squash commit title](https://github.com/refined-github/refined-github/issues/276) and [updates the PR’s title to match the commit title, if changed.](https://user-images.githubusercontent.com/1402241/51669708-9a712400-1ff7-11e9-913a-ac1ea1050975.png)
-- [](# "update-pr-from-base-branch") [Adds a button to update a PR from the base branch to ensure it builds correctly before merging the PR itself.](https://user-images.githubusercontent.com/1402241/106494023-816d9a00-647f-11eb-8cb1-7c97aa8a5546.png) GitHub only adds it when the base branch is "protected".
+- [](# "update-pr-from-base-branch") [Adds an "Update branch" button to every PR.](https://user-images.githubusercontent.com/1402241/234483592-4867cb2e-21cb-436d-9ea0-aedadf834f19.png) GitHub has the same feature, but it must be manually configured with protected branches.
```

Comment:
The tooltip is actually slightly pushed to the left, not aligned right. I only did it manually here to make it look better.

It can't be centered because it's cut off on sidebar-less views. I tried `tooltipped-e` but it looked bad too

<img width="670" alt="Screenshot 12" src="https://user-images.githubusercontent.com/1402241/234484416-2e661b24-088e-4845-a292-5c417b50c741.png">

Reply:
The tooltip was added to clarify that this is a RGH feature, because it blends in way too well otherwise.